### PR TITLE
feat: make oidc configurations profile compatible

### PR DESCRIPTION
**Pull request by danipran**
Also rename the environment variables for OIDC configuration.

Refs: PS-166

### CODE REVIEW

**Comment by terovirtanen** [comment]
**PUISTOTALKOOT-API branch is deployed to platta: https://puistotalkoot-pr169.api.dev.hel.ninja** :rocket::rocket::rocket:

File: haravajarjestelma/settings.py [hunk 1]
```diff
@@ -206,11 +206,13 @@
...
-    "ISSUER": env.str("TOKEN_AUTH_AUTHSERVER_URL"),
-    "API_AUTHORIZATION_FIELD": env.str("TOKEN_AUTH_FIELD_FOR_CONSENTS"),
-    "REQUIRE_API_SCOPE_FOR_AUTHENTICATION": env.bool("TOKEN_AUTH_REQUIRE_SCOPE_PREFIX"),
+    "AUDIENCE": env.str("OIDC_AUDIENCE"),
+    "API_SCOPE_PREFIX": env.str("OIDC_API_SCOPE_PREFIX"),
+    "ISSUER": env.str("OIDC_ISSUER"),
+    "API_AUTHORIZATION_FIELD": env.str("OIDC_API_AUTHORIZATION_FIELD"),
+    "REQUIRE_API_SCOPE_FOR_AUTHENTICATION": env.bool(
+        "OIDC_REQUIRE_SCOPE_FOR_AUTHENTICATION"
+    ),
```
[the remark this answers is not in the document]
Comment (danipran) [reply haravajarjestelma/settings.py:215]: Whoops! Right you are.

[the remark this answers is not in the document]
Comment (danipran) [reply haravajarjestelma/settings.py]: Fixed.

**Review by charn — APPROVED** [review]
LGTM! :+1: :shipit:

**Comment by terovirtanen** [comment]
**PUISTOTALKOOT-API branch is deployed to platta: https://puistotalkoot-pr169.api.dev.hel.ninja** :rocket::rocket::rocket:

**Pull request merged**